TESTING.md

```diff
@@ -109,6 +109,35 @@ mvn clean install -pl janusgraph-es -Delasticsearch.docker.image=elasticsearch
 mvn clean install -pl janusgraph-es -Delasticsearch.docker.version=1.5.1 -Delasticsearch.docker.image=elasticsearch
 ```
 
+### Running CQL Tests with a Cassandra
+
+**Note** Running Cql tests requires docker.
+
+```bash
```

Comment:
It would be good to have at least one sentence of explanation before each of this code blocks. Also, why are those two lines put together? Am I supposed to execute the CQL tests first for Cassandra 2 and then again for Cassandra 3?